src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -161,11 +161,9 @@ protected function writePrompt(OutputInterface $output, Question $question)
      */
     protected function writeError(OutputInterface $output, \Exception $error)
     {
-        if (null !== $this->getHelperSet() && $this->getHelperSet()->has('formatter')) {
-            $message = $this->getHelperSet()->get('formatter')->formatBlock($error->getMessage(), 'error');
-        } else {
-            $message = '<error>'.$error->getMessage().'</error>';
-        }
+        $formatter = new FormatterHelper();
```

Comment:
same ... could be a optional constructor arg, assuming one uses a custom implementation today.